File: Code/NewsWebsiteBackEnd/DTO/User/UserDetailsViewModel.cs
namespace NewsWebsiteBackEnd.DTO.User
{
    public class UserDetailsViewModel
    {
        public string Id { get; set; }
        public string UserName { get; set; }
        public string Email { get; set; }
        public string FullName { get; set; }
        public string? Country { get; set; }
        public string? ProfileImagePath { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public bool IsBlocked { get; set; }
        public string WebsiteLanguage { get; set; }
        public int UserTypeId { get; set; }
        public string UserTypeName { get; set; }
    }
}

File: Code/NewsWebsiteFrontEnd/src/app/models/user/user-details-view-model.ts
export interface UserDetailsViewModel {
  id: string;
  userName: string;
  email: string;
  fullName: string;
  country?: string;
  profileImagePath?: string;
  createdAt: Date;
  updatedAt?: Date;
  isBlocked: boolean;
  websiteLanguage: string;
  userTypeId: number;
  userTypeName: string;
}
